com.unity.render-pipelines.core/ShaderLibrary/Color.hlsl

```diff
@@ -254,7 +254,11 @@ real YCoCgCheckBoardEdgeFilter(real centerLum, real2 a0, real2 a1, real2 a2, rea
 }
 
 // Converts linear RGB to LMS
+#if defined(SHADER_API_SWITCH) // Must use full float precision on Switch to avoid artefact when using ACES tonemapping
```

Comment:
Should these just be float3s always, won't the fp16 be a problem on any platform that defines the real3 as fp16?